### ISSUE TITLE
Global settings error

### ISSUE THREAD

**Issue by giancastro** [geoserver/geoserver-cloud#245]
Every time that I try to open the global settings, an error arises. It happens using v1.0-RC20 or v1.0-RC21.

```
Can't instantiate page using constructor 'public org.geoserver.web.admin.GlobalSettingsPage()'. An exception has been thrown during construction!
```
![Screenshot from 2022-06-16 16-23-20](https://user-images.githubusercontent.com/16704806/174149380-f17b7903-f424-45e0-b0ce-14ea695e0379.png)

**Comment by bruschke** [comment]
Same issue

**Comment by groldan** [comment]
@giancastro @bruschke it's fixed now, can you test the latest `1.0-RC22`?

**Comment by giancastro** [comment]
@groldan , it's working now. Thanks!
